interfaces/repo.go

```diff
@@ -516,3 +516,126 @@ func (r *Repository) securitySnippetsForSnap(snapName string, securitySystem Sec
 	}
 	return snippets, nil
 }
+
+type addSnapError struct {
```

Comment:
Applied in 08cb524300f33aa952c5077e321996cc3a2b4f31

This will swallow errors if plug and slot have the same name and both fail.